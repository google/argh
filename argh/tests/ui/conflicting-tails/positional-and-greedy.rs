/// Command
#[derive(argh::FromArgs)]
struct Cmd {
    #[argh(positional)]
    /// positional
    positional: Vec<String>,

    #[argh(positional, greedy)]
    /// remainder
    remainder: Vec<String>,
}

fn main() {}
