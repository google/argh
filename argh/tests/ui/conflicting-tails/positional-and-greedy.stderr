error: Only the last positional argument may be `Option`, `Vec`, or defaulted.
 --> tests/ui/conflicting-tails/positional-and-greedy.rs:4:5
  |
4 |     #[argh(positional)]
  |     ^

error: Later positional argument declared here.
 --> tests/ui/conflicting-tails/positional-and-greedy.rs:8:5
  |
8 |     #[argh(positional, greedy)]
  |     ^
